src/lib/Recorder.ts

```diff
@@ -1,55 +1,64 @@
+/* eslint-disable max-statements */
+/* eslint-disable max-len */
```

Comment:
no eslint exceptions please, unless it's for a very specific reason and then it should be line-based (like no-console).
fix the code or provide a strong motivation for adjusting the rule at project-level.